### PR TITLE
Add document on github administration

### PR DESCRIPTION
**Pull request by dschuff**
As discussed in #1215

### CODE REVIEW

**Review by dtig — APPROVED** [review]

File: process/github.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- A few repositories with shared code projects
+    - [binaryen](https://github.com/WebAssembly/binaryen): Optimizer and toolchain library
+    - [wabt](https://github.com/WebAssembly/wabt): Low-level wasm binary toolkit
+    - [wasi-libc](https://github.com/WebAssembly/wasi-libc): C library implementation on WASI
+    - And a few other projects and test suites
+  
+# Administration
+The GitHub organization is managed by the CG and subject to the same
+decision-making processes as proposals or other CG activity.  It has several
+organization owners (for redundancy and to cover several time zones): this is
```
Comment (conrad-watt) [inline process/github.md:29]:
```suggestion
organization owners (for redundancy and to cover several time zones): these are
```

grammar nit

**Review by conrad-watt — APPROVED** [review]
LGTM

on what timescale are we planning to enact this - do we need a CG announcement?

**Comment by dschuff** [comment]
> on what timescale are we planning to enact this - do we need a CG announcement?

I guess I don't have a particular plan for the timeline, it was sufficiently non-urgent that it took me quite a while to get around to it :)
I think it's worth at least announcing in the CG, since probably some people who had commit rights in various repos will lose them, and need to ask to get them back; so they (and the folks they will need to ask) should be aware.
We can probably mitigate some disruption when we implement this by retaining existing commit privileges for folks who have frequently and/or recently committed (especially for non-proposal repos).

**Comment by dschuff** [comment]
As discussed and announced, I'm going to start working on implementing this. If there are issues, let me know (and if it's not clear from this PR what to do, let's fix that!)

**Pull request merged**